src/hpc/atomic.h

```diff
@@ -13,6 +13,7 @@
 #else
 
 #ifndef WARD_ENABLED
+#define AO_AO_TS_T 0    // HACK
```

Comment:
Could you atleast comment what this 'hack' is for? my googling skills find nothing.